Directory.Build.props

```diff
@@ -9,4 +9,9 @@
     <ItemGroup>
         <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta009" PrivateAssets="all" />
     </ItemGroup>
+    <PropertyGroup>
```

Comment:
Need to sign all assemblies as strongly typed assembly can be marked as InternalsVisibleTo only to other strongly typed assembly